guides/v2.1/config-guide/multi-site/ms_over.md

```diff
@@ -34,13 +36,15 @@ This section discusses the tasks required to use the `MAGE_RUN_TYPE` and `MAGE_R
 
 We use the following terms:
 
-*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites. 
+*	*Website* is the top-level container for sites, shipping methods, payment methods, and so on. To create completely separate sites that do not share cart, shipping methods, and so on,  you must create separate websites.
+
+	Using  Website Customer account can be shared with different Website from a single magento instance. One Website contains at least one store. Catalog Prices should be manage at Website level.
 
 *	*Store* is contained by a {% glossarytooltip a3c8f20f-b067-414e-9781-06378c193155 %}website{% endglossarytooltip %}. In turn, a store contains at least one *store view*. 
 
-	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures. 
+	Multiple stores can share cart, user sessions, payment gateways, and so on, but they have separate catalog structures and Catalog price and Catalog Qty can't be manage at store level.
 
-	Store views change the way pages are presented, and are typically used to display a store with different layouts or languages. 
+	Store views change the way pages are presented, and are typically used to display a store with different layouts or languages.You can manage different currency at each store views also.
```

Comment:
@RakeshJesadiya Please add one space after the colon.

Reply:
Also reword it to be correct, like "You can manage different currencies per store view."

Reply:
Done.